OpenRA.Mods.Common/Traits/Render/LeavesTrails.cs

```diff
@@ -125,7 +125,7 @@ public void Tick(Actor self)
 				cachedFacing = facing != null ? facing.Facing : 0;
 				ticks = 0;
 
-				cachedInterval = isMoving && !wasStationary ? Info.MovingInterval : Info.StationaryInterval;
+				cachedInterval = isMoving ? Info.MovingInterval : Info.StationaryInterval;
```

Comment:
This unconditionally overrides the `cachedInterval = Info.StartDelay` above, breaking that feature.